pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.mypy]
+python_version = "3.10"
+show_error_codes = true
+check_untyped_defs = true
+warn_unused_configs = true
+follow_imports = "skip"
+ignore_missing_imports = true
+
+[tool.black]
+skip-string-normalization = true
+
+[tool.flake8]
+show-source = true
+max-line-length = 99
+application-import-names = "wazo_agid"
+unused-arguments-ignore-abstract-functions = true
```

Comment:
Why we want to add this new config? We should fix the warning, no?

Reply:
I thought you wanted me to keep the existing options? I can certainly remove it, but from the name I thought it was innocent. I don't usually use flake8, but it is normal to not "use" arguments in abstract methods, so if it were to flag it as invalid I could see how that would be a problem.

Reply:
I didn't read what it was. But we hadn't this option before :laughing:

Reply:
Hmm, true... I don't know where it came from then, sorry. I'll remove it and we can add it, if it comes up later.